src/views/RequestTypeForm/index.test.tsx

```diff
@@ -242,7 +251,46 @@ describe('The request type form page', () => {
       }
     };
 
-    renderPage([intakeMutation, intakeQuery({ requestType: 'MAJOR_CHANGES' })]);
+    renderPage([
+      intakeMutation,
+      intakeQuery({ requestType: 'MAJOR_CHANGES' }),
+      {
+        request: {
+          query: GetGovernanceTaskListQuery,
+          variables: {
+            id: INTAKE_ID
+          }
+        },
+        result: {
+          data: {
+            systemIntake: {
+              __typename: 'SystemIntake',
+              id: INTAKE_ID,
+              requestName: 'Mock system intake',
+              itGovTaskStatuses: {
+                __typename: 'ITGovTaskStatuses',
+                intakeFormStatus: ITGovIntakeFormStatus.READY,
+                feedbackFromInitialReviewStatus: ITGovFeedbackStatus.CANT_START,
+                decisionAndNextStepsStatus: ITGovDecisionStatus.CANT_START,
+                bizCaseDraftStatus: ITGovDraftBusinessCaseStatus.CANT_START,
+                grtMeetingStatus: ITGovGRTStatus.CANT_START,
+                bizCaseFinalStatus: ITGovFinalBusinessCaseStatus.CANT_START,
+                grbMeetingStatus: ITGovGRBStatus.CANT_START
+              },
+              step: SystemIntakeStep.INITIAL_REQUEST_FORM,
+              state: SystemIntakeState.OPEN,
+              decisionState: SystemIntakeDecisionState.NO_DECISION,
+              governanceRequestFeedbacks: [],
+              submittedAt: null,
+              updatedAt: null,
+              grtDate: null,
+              grbDate: null,
+              businessCase: null
+            }
+          }
+        }
+      }
+    ]);
```

Comment:
We have mocked queries in src/data/mock/systemIntake.ts that can be used here.

The `intakeQuery` function can be replaced with `getSystemIntakeQuery`, and the mocked query for `GetGovernanceTaskListQuery` can be replaced with `getGovernanceTaskListQuery()`.

Reply:
Can we just leave this the way it is for test data? It's easier to get the context of what is intended for the specific tests this way.

Reply:
Leaving this block the way it is.

The mock functions aren't generalized enough to be swapped in place. The adjustments needed here wouldn't be much, but I think that type of work is separate. Since this task is just v1 cleanup work, I think this patch is good enough with this mix of old and new code.

There is a fe sync on mock data usage coming up where we can all share what we think is most effective.

I don't mind using mock functions when their arguments and returns all follow a similar structure, and I can think about them less each time I used them, but at the moment they are slightly different enough that they need to be looked into to see if you'll get the data back that you need. I don't mind that work if the task at hand involves a lot of tests, but for a few changes I think literals on the spot are reasonable.

Reply:
> The mock functions aren't generalized enough to be swapped in place

I'm not sure what you mean here because they can be swapped out as-is. The full intake query can be replaced with `getGovernanceTaskListQuery()`. The `intakeQuery` function is the same as `getSystemIntakeQuery` with the same function arguments but no typing. The only other thing that would need to be changed to make this work is the `INTAKE_ID`. We can talk about your other points during the mock data sync, but just wanted to point that out.

Approving this with the understanding that this test will be revisited if needed after the mock data sync.